lib/Auth/Process/SwitchAuth.php

```diff
@@ -210,6 +214,7 @@ private function getMFAForUid($state)
         $result = [];
         if (!empty($state['Attributes'][self::MFA_TOKENS])) {
             foreach ($state['Attributes'][self::MFA_TOKENS] as $mfaToken) {
+                $mfaToken = json_decode($mfaToken, true);
```

Comment:
```suggestion
                if (is_string($mfaToken)) {
                    $mfaToken = json_decode($mfaToken, true);
                }
```

This has changed in a breaking change release (9.0.0), this filter now requires the tokens to be objects. If you want to keep support for strings, make it conditional.